web-frontend/src/store.ts

```diff
@@ -16,17 +16,10 @@ export const isTestEnvironment = readable(
     window.location.hostname.startsWith('meta.dev'),
 );
 
-export function baseUrl() {
```

Comment:
Why was it removed? AFAIR In old setup it was required for working in between different environments. There are no tests here, so could you confirm that mets is working as expected?

Reply:
It essentially does not do anything useful, I am not sure why it used to be necessary.

If `fetch` requests start with a `/` then the url will result in the exact same `protocol://host:port` as this function is calculating.

Speaking about tests... I have not found any js test and also no setup for running js tests in the CI. Did I miss anything?

Reply:
There is no tests at all on the FE side, this is what I meant.

Reply:
Adding a test setup is beyond the scope of this PR.

Reply:
Great, because I didn't event think about it to add it in this PR, neither proposed it.